### ISSUE TITLE
Crashes not uploaded for Android and IOS symbols not loaded for UE5.1 crashes

### ISSUE THREAD

**Issue by gamerdoku** [BugSplat-Git/bugsplat-unreal#60]
Hi,

I integrated version 0.2.5 of BugSplat Unreal plugin in UE5.1. 
I see crashes and symbols correctly uploaded and parsed for Linux server builds. But I am not seeing the crash at all reported for Android. 
For iOS I see the crash but it is not symbolicated.

All the settings are standard. I am currently testing with development builds. 

I have tried global as well as Project specific configs to try to make it work.

I am seeing Android symbol upload process start as part of packaged builds but never completes. No error in the logs either.
Is Android symbol upload supported when running UE Editor on Mac?

I see IOS symbols upload successfully.

I do not see any logs for crash being sent from Android client run.

**Comment by bobbyg603** [comment]
Hi @gamerdoku 👋,

Can you please DM the name of you database to me on Discord, or reach out via the bugsplat.com support tool?

For Android, what device/version are you using? Are you testing on hardware or in an emulator? Android symbol uploads are supported from macOS, but depending on your network connection, they might take a long time.

**Comment by gamerdoku** [comment]
I have shared the requested database name.

For Android I am testing on One Plus 8 Pro. 

Android symbols definitely are failing for me on Mac with the following errors:

```
BugSplat postprocessing: Start debug symbols upload for iOS
UATHelper: Packaging (Android (ASTC)): BugSplat postprocessing: Unexpected platform Android. Terminating...
```
On Windows host I am not seeing an error but do not see a success message either.

**Comment by bobbyg603** [comment]
The iOS issue is mostly fixed... there's still a UX issue https://github.com/BugSplat-Git/BugSplat-iOS/issues/11

Android Symbol Upload issue should be resolved by https://github.com/BugSplat-Git/bugsplat-unreal/issues/61 and a change to increase the timeout on our backend.

Haven't had a chance to dive into the Android crash issue yet but that's next...

**Comment by bobbyg603** [comment]
Android crash issue was fixed by #77
